ckanext/canada/tables/travelq.yaml

```diff
@@ -187,37 +187,189 @@ resources:
   - datastore_id: destination_en
     datastore_type: text
     label:
-      en: Places visited (English)
+      en: Place visited (English)
       fr: Endroits visités (anglais)
     description:
-      en: To include names of all places visited during the trip in question, in English
+      en: To include name of the place visited during the trip in question, in English.
       fr: Cette zone indique, en anglais, le nom de tous les endroits visités pendant le voyage.
+    form_help:
+      en: Note, use the format {City Name}, {Country Name} (e.g. Ottawa, Canada or New York City, United States of America)
+      fr: Note, use the format {City Name}, {Country Name} (e.g. Ottawa, Canada or New York City, United States of America)
     obligation: Mandatory
     excel_required: true
+    # Dev NOTE: excel_error_formula: Excel and Open Office Calc parity issues with REGEX. REGEX does not exist in Excel, and sheetname references in OOC work differently.
```

Comment:
open office calc? I've found the sheet references that recombinant generates e.g. for validation and links work fine